src/core/grid.rs

```diff
@@ -1,16 +1 @@
-use std::error::Error;
```

Comment:
was this already orphaned code? (i assume so)

Reply:
Yea, it mostly just sat there throughout our branches as a legacy of the old setup, pre shapes.